src/helperFunctions/install.py

```diff
@@ -242,8 +242,12 @@ def check_distribution():
     if distro.id() == 'fedora':
         logging.debug('Fedora detected')
         return 'fedora'
-    logging.critical('Your Distribution ({} {}) is not supported. FACT Installer requires Ubuntu 18.04, 20.04 or compatible!'.format(distro.id(), distro.version()))
-    sys.exit(1)
+    msg = f'Your Distribution ({distro.id()} {distro.version()}) is not supported. FACT Installer requires Ubuntu 18.04, 20.04 or compatible!'
+    if allow_unsupported:
+        logging.warning(msg + '(Ignoring)')
```

Comment:
IMHO we can omit the "ignoring" and even convert it to log level "info" so it does not get mixed up with actual warnings or errors that might occur during installation and which we might want to report to the user (also for FACT docker)

Reply:
Done.